srml/support/procedural/src/storage/transformation.rs

```diff
@@ -675,6 +675,7 @@ fn store_functions_to_metadata (
 		let def_get = quote! {
 			pub struct #struct_name<#traitinstance>(pub #scrate::rstd::marker::PhantomData<#traitinstance>);
 			#[cfg(feature = "std")]
+			#[allow(non_upper_case_globals)]
```

Comment:
I think that is not required anymore? As you already made the name upper case?

Reply:
the name is not fully upper case, I wanted at first to do the switch snake - cammel, but thinking twice it looked like another potential name collision issue.